upup/pkg/fi/lifecycle.go

```diff
@@ -30,6 +31,12 @@ const (
 
 	// LifecycleExistsAndWarnIfChanges will check that the task exists and will warn on changes, but then ignore them
 	LifecycleExistsAndWarnIfChanges Lifecycle = "ExistsAndWarnIfChanges"
+
+	// LifecycleValidationLoose allows for validiation to warn and pass. This is optional.
+	LifecycleValidationLoose LifecycleValidationPolicy = "loose"
```

Comment:
Is there a reason not to use the `Lifecycle` values here?  (Other than you hate the names?)  Doesn't strict=LifecycleExistsAndValidates and loose=ExistsAndWarnIfChanges ?